fontforgeexe/fontinfo.h

```diff
@@ -0,0 +1,21 @@
+#ifndef FONTFORGE_FONTFORGEEXE_FONTINFO_H
+#define FONTFORGE_FONTFORGEEXE_FONTINFO_H
+
+#include <ggadget.h>
```

Comment:
It's about search paths. When looking for a bracketed header name, gcc skips those provided by `-I` unless they trail a `-I-`. So anything not installed at compile time gets referenced with quotes.